src/tabrmd-init.c

```diff
@@ -99,6 +99,12 @@ gmain_data_cleanup (gmain_data_t *data)
     if (data->loop != NULL) {
         main_loop_quit (data->loop);
     }
+    if (data->tpm2 != NULL) {
+        if (data->tpm2->sapi_context != NULL) {
+            g_free(data->tpm2->sapi_context);
```

Comment:
nit: Should there be a space here?

```suggestion
            g_free (data->tpm2->sapi_context);
```